system_metrics_collector/CMakeLists.txt

```diff
@@ -49,6 +49,8 @@ add_library(system_metrics_collector SHARED
   src/system_metrics_collector/proc_cpu_data.cpp
   src/system_metrics_collector/proc_pid_cpu_data.cpp
   src/system_metrics_collector/utilities.cpp
+  src/topic_statistics_collector/received_message_period.hpp
```

Comment:
Why include these headers for the build? If nothing includes them, then they shouldn't be built. The tests include them, so their build is tested.

Reply:
Will remove